packages/create-cloudflare/src/helpers/codemod.ts

```diff
@@ -76,3 +77,31 @@ export const transformFile = (
 		writeFile(filePath, recast.print(ast).code);
 	}
 };
+
+export const loadSnippets = (snippetsPath: string) => {
+	if (!existsSync(snippetsPath)) {
+		return null;
+	}
+
+	if (!lstatSync(snippetsPath).isDirectory) {
+		return null;
+	}
+
+	const files = readdirSync(snippetsPath);
+
+	return (
+		files
+			// don't try loading directories
+			.filter((fileName) => lstatSync(join(snippetsPath, fileName)).isFile)
+			// only load js or ts files
+			.filter((fileName) => [".js", ".ts"].includes(extname(fileName)))
+			.reduce((acc, snippetPath) => {
+				const [file, ext] = snippetPath.split(".");
+				const key = `${file}${ext === "js" ? "Js" : "Ts"}`;
+				return {
+					...acc,
+					[key]: parseFile(join(snippetsPath, snippetPath))?.body,
+				};
+			}, {}) as Record<string, recast.types.ASTNode[]>
+	);
```

Comment:
NIT - personally I find use of filters and reduce actually makes the code less readable and in fact less performant due to the recreation of arrays and objects on each iteration.

How about:

```ts
	const result: Record<string, Program["body"]|undefined> = {}
	for(const file of files) {
		if (!lstatSync(join(snippetsPath, file)).isFile) {
		  continue;
		}
		const ext = extname(file);
		if (!['.js', '.ts'].includes(ext)) {
			continue;
		}
		const baseName = basename(ext);
		const key = `${baseName}${ext === "js" ? "Js" : "Ts"}`;
		result[key] = parseFile(join(snippetsPath, file))?.body;
	}

	return result;
```